src/components/Lesson/LessonButton.js

```diff
@@ -0,0 +1,34 @@
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import styles from './Lesson.scss';
+import {getTranslator} from '../../selectors/translate';
+import {getReadmepathFromLessonpath} from '../../util';
+import InstructionButton from './InstructionButton';
+
+const LessonButton = ({path, lessons, t}) => {
```

Comment:
Jeg synes input props her burde være `buttonPath` og `t`, og at logikken for buttonPath skal legges ned i mapStateToProps.

Dette gjelder for både LessonButton og ReadmeButton.